src/exporter.js

```diff
@@ -52,7 +44,7 @@ function exporter (hash, dagService, options, callback) {
         rs.push(unmarshaledData.data)
         rs.push(null)
       }
-      ee.emit('file', { stream: rs, path: name, dir: dir })
+      this.push({ stream: rs, path: name, dir: dir })
       if (callback) {
         callback()
       }
```

Comment:
at the top of the function do `if (!callback) { callback = function noop () {} }` and replace these with just `callback`